src/PlaneWaveBasis.jl

```diff
@@ -470,3 +480,65 @@ function PlaneWaveBasis(basis::PlaneWaveBasis; use_symmetry)
                                [[identity_symop()] for _ in 1:length(kcoords)];
                                fft_size=basis.fft_size)
 end
+
+
+"""
+Gather the distributed k-Point data on the master process and return
+it as a `PlaneWaveBasis`. On the other (non-master) processes `nothing` is returned.
+The returned object should not be used for computations and only to extract data
+for post-processing and serialisation to disk.
+"""
+function mpi_kgather(basis::PlaneWaveBasis)
```

Comment:
I'd rather we keep the mpi_* for actual MPI methods. `gather_kpts` (consistent with comm_kpts) or something?

Reply:
This file is much too long already, would it make sense to move these methods elsewhere?

Reply:
gather_kpts is good. I'll think about the splitup.